deny.toml

```diff
@@ -73,6 +73,12 @@ ignore = [
     #
     # TODO: Upgrade clap to v4.x.
     "RUSTSEC-2021-0145",
+    # Ignore RUSTSEC-2024-0421 because the licenses of some dependencies in
```

Comment:
I'm not sure about the full implications of ignoring this vulnerability. Perhaps we should address it in a separate PR to allow for further discussion, just in case?

Reply:
Yep, it's better to update the url crate to 2.5.4 to fix this warning.

Pls wait https://github.com/tikv/tikv/pull/17963 merged.